### PR TITLE
This is a product restriction

### PR DESCRIPTION
**Pull request by ManuelGalindo**
The Workloads cant be accessible due the private API address of the AKS Private Cluster

### CODE REVIEW

**Comment by PRMerger8** [comment]
@ManuelGalindo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/process/environments.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ All  environments targeted by deployment jobs of a specific run of a pipeline ca
 
   > [!div class="mx-imgBorder"]
   > ![Environments in run details](media/environments-run.png)
+  
+Note: If you are using an AKS Private Cluster the Worloads tab is not aviliable
```
Comment (juliakm) [inline docs/pipelines/process/environments.md:103]:
```suggestion
If you are using an AKS Private Cluster, the **Workloads** tab is not available
```

**Review by juliakm — CHANGES_REQUESTED** [review]

**Comment by juliakm** [comment]
#sign-off

**Comment by PRMerger13** [comment]
@juliakm : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**